Emojigrams/Emojigrams/EmojigramTitleScreenViewController.swift

```diff
@@ -11,8 +11,9 @@ import UIKit
 class EmojigramTitleScreenViewController: UIViewController {
     let emojigramStore = EmojigramStore()
     
-    override func viewDidLoad() {
```

Comment:
#? how come are we getting rid of viewDidLoad? is viewWillAppear a better option for loading issues?

Reply:
`viewDidLoad` is called when the VC is initially loaded into memory and won't be called again until a new instance is created. You'd want `viewWillAppear` when the page might be viewed multiple times but only initialized once. In this case, I want to hide the navigation bar on the TitleScreen but show it on other pages. Since the titleScreen is the root viewController in the navigationStack, when the user goes back from either of the guessing screens or the emojigram creation screen, we need viewWillAppear to hide the navBar each time whereas viewDidLoad would only hide the bar the very first time.